util/drakeGradientUtil.cpp

```diff
@@ -141,11 +144,13 @@ void setSubMatrixGradient(Eigen::MatrixBase<DerivedA>& dM, const Eigen::MatrixBa
 }
 
 // explicit instantiations
```

Comment:
I estimate I spent about 20% of my time figuring out which explicit instantiations I need and adding them to the list.